openmetadata-ui/src/main/resources/ui/src/pages/TagPage/TagPage.tsx

```diff
@@ -0,0 +1,305 @@
+          tagItem.fullyQualifiedName ?? '',
+          tab
+        ),
+      });
+    }
+  };
+
+  const fetchClassificationTagAssets = async () => {
+    try {
+      const encodedFqn = getEncodedFqn(escapeESReservedCharacters(tagFqn));
+      const res = await searchData(
+        '',
+        1,
+        0,
+        `(tags.tagFQN:"${encodedFqn}")`,
+        '',
+        '',
+        SearchIndex.ALL
+      );
+
+      setAssetCount(res.data.hits.total.value ?? 0);
+    } catch (error) {
+      setAssetCount(0);
+    }
+  };
+
+  const handleAssetSave = useCallback(() => {
+    fetchClassificationTagAssets();
+    assetTabRef.current?.refreshAssets();
+    tab !== 'assets' && activeTabHandler('assets');
+  }, [assetTabRef]);
+
+  const tabItems = useMemo(() => {
+    const items = [
+      {
+        label: <div data-testid="overview">{t('label.overview')}</div>,
+        key: 'overview',
+        children: (
+          <Row>
+            <Col className="description-content" span={24}>
+              <DescriptionV1
+                description={tagItem?.description}
+                entityFqn={tagItem?.fullyQualifiedName}
+                entityName={getEntityName(tagItem)}
+                entityType={EntityType.GLOSSARY_TERM}
+                showActions={!tagItem?.deleted}
+              />
+            </Col>
+          </Row>
+        ),
+      },
+      {
+        label: (
+          <div data-testid="assets">
+            {t('label.asset-plural')}
+            <span className="p-l-xs ">
+              {getCountBadge(assetCount ?? 0, '', activeTab === 'assets')}
+            </span>
+          </div>
+        ),
+        key: 'assets',
+        children: (
+          <AssetsTabs
+            assetCount={assetCount}
+            entityFqn={tagItem?.fullyQualifiedName ?? ''}
+            isSummaryPanelOpen={Boolean(previewAsset)}
+            permissions={MOCK_GLOSSARY_NO_PERMISSIONS}
+            ref={assetTabRef}
+            onAddAsset={() => setAssetModalVisible(true)}
+            onAssetClick={handleAssetClick}
+            onRemoveAsset={handleAssetSave}
+          />
+        ),
+      },
+    ];
+
+    return items;
+  }, [tagItem, activeTab, assetCount, handleAssetSave]);
+
+  const icon = useMemo(() => {
+    if (tagItem?.style && (tagItem as Tag).style?.iconURL) {
+      return (
+        <img
+          className="align-middle object-contain"
+          data-testid="icon"
+          height={36}
+          src={(tagItem as Tag).style?.iconURL}
+          width={32}
+        />
+      );
+    }
+
+    return;
+  }, [tagItem]);
+
+  useEffect(() => {
+    getTagData();
+    fetchClassificationTagAssets();
+  }, []);
+
+  useEffect(() => {
+    if (tagItem) {
+      handleBreadcrumb(
+        tagItem.fullyQualifiedName ? tagItem.fullyQualifiedName : tagItem.name
+      );
+    }
+  }, [tagItem]);
+
+  return (
+    <>
+      {isLoading ? (
+        <Loader />
+      ) : (
+        tagItem && (
```

Comment:
tagItem must be there if loader is done

Reply:
Updated

Reply:
The properties of tagItem are giving TS errors if we will remove that.

Reply:
which type of ts errors